src/utils/dexFormater.js

```diff
@@ -55,21 +60,33 @@ export const makeAssetToShow = (asset, symbol) => {
   return assetToShow;
 };
 
-export const formatProperlyValue = (asset, value, symbol, decimals) => {
+export const formatProperlyValue = (asset, value, decimals, symbol) => {
```

Comment:
Terrible name. Maybe `formatAssetsWithSymbol` and move to `walletHelpers`? Or even better. Change the formatAssets in walletHelpers to optionally accept symbol and add it if provided.